datalake_scripts/helper_scripts/output_builder.py

```diff
@@ -0,0 +1,53 @@
+class CsvBuilder:
+
+    @staticmethod
+    def create_csv(complete_response, atom_type):
+        complete_csv = ['hashkey,atom_type,atom_value_best_matching,atom_found,events_number,first_seen,last_updated,'
+                        'threat_types,ddos.score.risk,fraud.score.risk,hack.score.risk,leak.score.risk,'
+                        'malware.score.risk,phishing.score.risk,scam.score.risk,spam.score.risk,sources,tags,'
+                        'href_graph,href_history,href_threat,href_threat_webGUI']
+
+        for threat in complete_response.keys():
+            response = complete_response[threat]
+            if 'threat_found' in response.keys():  # means threat not found
+                line = f"{response['hashkey']},{atom_type},{threat},{False},{None},{None},{None},{None},{None},{None}" \
+                       f",{None},{None},{None},{None},{None},{None},{None},{None},{None},{None},{None},{None}"
```

Comment:
Might be rewritten as:
```suggestion
                line = f"{response['hashkey']},{atom_type},{threat},{False}," + ','.join([f"{None}"] * 18)
```